### DISCUSSION TITLE
Hello Data Nerds

### DISCUSSION THREAD

**Discussion by 3liud** [TechBlurbs/maandamano-mondays-sentiment-analysis#8] [Ideas]
I am sure you have all seen the new project. Let's get it rolling.

**Comment by VictorOmondiCDS** [comment]
Hmm, @3liud Just noticed I can't assign more than 1 person to an issue.
Since this project is private, only 1 person can be assigned to an issue
Check explanation here: https://stackoverflow.com/a/63091799/8797509
If we make it public, we can assign up to 10 people to an issue. I'll go ahead and make the project public

**Reply by VictorOmondiCDS** [reply]
I've made it public

**Comment by Stephen-Kamau** [comment]
@VictorOmondi1997 and @3liud  You could assign guys the issues or post it via whatsapp could so that it goes fast.
